fontc_crater/src/ci/html.rs

```diff
@@ -231,7 +231,7 @@ fn make_table_body(runs: &[RunSummary]) -> Markup {
     let iter = runs.iter().enumerate().map(|(i, run)| {
         let prev = (i > 0).then(|| &runs[i - 1]);
         let is_last = i == runs.len() - 1;
-        let default_visibility = i >= runs.len() - 16;
```

Comment:
idk what this has to do with relative paths?

Reply:
yes it's an unrelated crash but was triggered here because I was testing in a new directory (and so this overflows)